final_prep/mod_zero_hero.rb

```diff
@@ -1,46 +1,67 @@
-
+hero_name = "Delivery-Boy Man"
+special_ability = "laser eye power"
 # Declare two variables - greeting AND catchphrase
+
+greeting = "It's me, #{hero_name}!"
+catchphrase = "Special delivery for Invader X, #{special_ability}!"
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
 
 # Declare two variables - power AND energy - set to integers
-
+power = 30
+energy = 100
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
-
+full_power = power * 500
+full_energy = energy + 150
 # Declare two variables - is_human and identity_concealed - assigned to booleans
-
+is_human = true
+identity_concealed = true
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
-
+arch_enemies = ["Invader X", "Ndnd", "Ghostware"]
+sidekicks = ["Captian Yesterday", "Cloberella", "Super King"]
 # Print the first sidekick to your terminal
-
+print sidekicks[0]
 # Print the last arch_enemy to the terminal
-
+print arch_enemies[2]
 # Write some code to add a new arch_enemy to the arch_enemies array
-
+arch_enemies = arch_enemies + ["The Zookeeper"]
 # Print the arch_enemies array to terminal to ensure you added a new arch_enemey
-
+print arch_enemies
 # Remove the first sidekick from the sidekicks array
-
+sidekicks.shift(1)
 # Print the sidekicks array to terminal to ensure you added a new sidekick
-
+print sidekicks
 # Create a function called assess_situation that takes three arguments - danger_level, save_the_day, bad_excuse
 #   - danger_level should be an integer
-#   - save_the_day should be a string a hero would say once they save the day 
+#   - save_the_day should be a string a hero would say once they save the day
 #   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
-
+puts "\n"
+puts "_" * 10
+
+def assess_situation(danger_level, save_the_day, bad_excuse)
+end
+danger_level = 41
+save_the_day = "All in a days work."
+bad_excuse = "That's a nope for me."
 # Your function should include an if/else statement that meets the following criteria
 #   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the bad_excuse to the terminal
 #   - Anything danger_level that is between 10 and 50 should result in printing the save_the_day string to the terminal
 #   - If the danger_level is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the terminal.
-
+  if danger_level >= 50
+    print bad_excuse
+  elsif danger_level.between?(10,50)
```

Comment:
Nice way to assess that, though you can also just opt to place the most complex logic in your `else` block.